pkg/config/expand.go

```diff
@@ -863,8 +863,8 @@ func expandSimpleVariable(context varContext, trackModuleGraph bool) (string, er
 
 		// ensure that the target module outputs the value in the root module
 		// state and not just internally within its deployment group
-		if !slices.Contains(toMod.Outputs, varRef.name) {
-			toMod.Outputs = append(toMod.Outputs, varRef.name)
+		if !slices.ContainsFunc(toMod.Outputs, func(o modulereader.OutputInfo) bool { return o.Name == varRef.name }) {
+			toMod.Outputs = append(toMod.Outputs, modulereader.OutputInfo{Name: varRef.name})
```

Comment:
Are you intentionally loosing `description` and `sensitive` here?

Reply:
Right now this block ends in an error. When we enable intergroup references, `Sensitive` will probably be set to a user-configurable global value for all automatically generated output values. This will probably be `true` by default.

In the TF writer, a default description is generated if the field is its zero value. This is, for example:

```hcl
output "internal_ip_simplevm" {
  description = "Generated output from module 'simplevm'"
  value       = module.simplevm.internal_ip
}
```

I don't think it's necessarily helpful to set a different default value here, but I could be convinced, esp. if we want to specifically call out that it was generated automatically by an intergroup reference.

Reply:
That is reasonable, thank you for explanation.